### PR TITLE
adding in a simpler animation and style changes for breadcrumbs

### PR DESCRIPTION
**Pull request by jivewise**
animation-prototype

Adds a small animation (and colors) to breadcrumb steps.

@fredasaurus: Thoughts on if it should be included in the codebase?

### CODE REVIEW

File: src/components/breadcrumbs.cjsx [hunk 1]
```diff
@@ -30,12 +30,16 @@ module.exports = React.createClass
...
         classes.push('active')
+        bsStyle = 'primary'
         # classes.push('disabled')
         title = "Current Step (#{step.type})"
 
       if step.is_completed
         classes.push('completed')
-        bsStyle = 'primary'
+        if step.correct_answer_id is step.answer_id or not step.correct_answer_id
+          classes.push('btn-success')
```
Comment (philschatz) [inline src/components/breadcrumbs.cjsx]: Should this just be `bsStyle = 'success'`? Is there a need for both `btn-primary` and `btn-success` to be on the element?

Comment (jivewise) [inline src/components/breadcrumbs.cjsx]: No, I just use that class in the css to get the correct color, for the current step.  Bad naming, I'll update to reflect it as we go further.

**Comment by jivewise** [comment]
Looking for feedback on this animation.  It's far simpler, in terms of code and the animation.  Do we want something more complex?

**Comment by philschatz** [comment]
Closing this for now. may reuse code once we start putting animations back in.

**Pull request closed without merging**